doc/articles/features/uno-material.md

```diff
@@ -18,21 +18,18 @@ For complete instructions on using Uno Material in your projects, including a se
                 <!-- Load WinUI resources -->
                 <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
+                <!-- Load Material Color Palette with optional ColorPaletteOverrideSource -->
```

Comment:
I think the ColorPaletteOverrideSource should be always be present in the docs because I think most people will actually want their own colors instead of the pre-defined ones.